options/locale/locale_en-US.ini

```
@@ -1782,6 +1782,8 @@ pulls.delete.text = Do you really want to delete this pull request? (This will p
 
 pulls.recently_pushed_new_branches = You pushed on branch <strong>%[1]s</strong> %[2]s
 
+pull.deleted_branch = <deleted>:%s
```

```suggestion
pull.deleted_branch = (deleted):%s
```

avoiding making it like a HTML tag.